roles/calibre-web/tasks/install.yml

```diff
@@ -42,21 +42,21 @@
 #  ignore_errors: True
 ##
 # Implementing this with Ansible command module for now.
-- name: Download Calibre-Web dependencies (using pip) into python2.7 virtual environment {{ calibreweb_venv_path }}
-  pip:
+- name: Download Calibre-Web dependencies (using pip) into python3.7 virtual environment {{ calibreweb_venv_path }}
+  pip3:
     requirements: "{{ calibreweb_venv_path }}/requirements.txt"
-    virtualenv: "{{ calibreweb_venv_path }}"    # /usr/local/calibre-web
+    virtualenv: "{{ calibreweb_venv_path }}"    # /usr/local/calibre-web-py3
     virtualenv_site_packages: no
-    virtualenv_command: /usr/bin/virtualenv
-    virtualenv_python: python2.7
+    virtualenv_command:  python3 -m venv {{ calibreweb_venv_path }}
+#    virtualenv_python: python3.7
   when: internet_available | bool
 
 # 2020-02-02: IS PYTHON 3 POSSIBLE ABOVE AND BELOW?
 
-- name: Symlink {{ calibreweb_venv_path }}/vendor -> {{ calibreweb_venv_path }}/lib/python2.7/site-packages - to keep {{ calibreweb_venv_path }}/cps.py happy
+- name: Symlink {{ calibreweb_venv_path }}/vendor -> {{ calibreweb_venv_path }}/lib/python2.7/site-packages - to keep {{ calibreweb_venv_path }}/cps.py happy?
   file:
-    src: "{{ calibreweb_venv_path }}/lib/python2.7/site-packages"
-    path: "{{ calibreweb_venv_path }}/vendor"    # /usr/local/calibre-web
+    src: "{{ calibreweb_venv_path }}/lib/python3.7/site-packages"
```

Comment:
Ubuntu 18.04 uses Python 3.6.x  — Whereas the above line mentions 3.7 — how should/would we accommodate OS's that do not offer 3.7.x ?

(Ubuntu 20.04 arriving soon may include Python 3.8.x, as another example.)

Reply:
get upstream to correct the layout as not to need vendor.